src/modules/twap/containers/TwapFormWidget/index.tsx

```diff
@@ -101,10 +103,14 @@ export function TwapFormWidget() {
           deadline={deadlineState}
           items={orderDeadlines}
           setDeadline={(value) => updateSettingsState(value)}
-          labels={LABELS_TOOLTIPS.totalDuration}
+          label={LABELS_TOOLTIPS.totalDuration.label}
+          tooltip={renderTooltip(LABELS_TOOLTIPS.totalDuration.tooltip, {
+            parts: numberOfPartsValue,
+            partDuration: timeInterval,
+          })}
```

Comment:
Ok, that's the second place where you explicitly have a tooltip fn, but then, why not just call the fn here?